src/main/java/io/sentry/cli/SentryCliRunner.java

```diff
@@ -92,6 +94,17 @@ public SentryCliRunner(
     }
   }
 
+  private @Nullable String escape(final @Nullable String toEscape, final boolean isWindows) {
+    if (toEscape == null) {
+      return null;
+    }
+    if (isWindows) {
+      return toEscape.replaceAll(" ", "^ ");
+    } else {
+      return toEscape.replaceAll(" ", "\\ ");
```

Comment:
Looks like `\\` gets swallowed by the regex pattern, so to actually get a `\` into the commandline we would have to replace whitespace with `\\\\`